docs/content/preview/releases/release-notes/v2.20.md

```diff
@@ -28,15 +28,32 @@ What follows are the release notes for the YugabyteDB v2.20 release series. Cont
 
 For an RSS feed of all release series, point your feed reader to the [RSS feed for releases](../../index.xml).
 
-{{< warning title="Technical Advisory: Use database version 2.20.1.3 or later" >}}
-YugabyteDB releases in the v2.20 series prior to v2.20.1.3 contain an issue where an index update can be wrongly applied leading to index corruption for the following rare cases:
+{{< warning title="Technical Advisories" >}}
+
+<br>**Index update can be wrongly applied**
+
+Impacts: v2.20.0.0 to 2.20.1.2
+
+The impacted releases contain an issue where an index update can be wrongly applied, leading to index corruption in the following rare cases:
 
 - Update to a table with non-unique indexes and self-referential triggers where both the index updates, and the trigger application affect the same rows.
 - Multi-row update to a table with a non-unique partial index.
 
 For more information, see [#20648](https://github.com/yugabyte/yugabyte-db/issues/20648).
 
-If you're using a previous v2.20 database version, it is recommended that you upgrade to YugabyteDB 2.20.1.3 or later.
+If you're using a v2.20 series release prior to v2.20.1.3, it is recommended that you upgrade to YugabyteDB 2.20.1.3 or later.
+
+**Correctness issue for queries using SELECT DISTINCT**
+
+Impacts: v2.20.1.0 to 2.20.1.3
+
+The impacted releases contain a correctness issue for queries using SELECT DISTINCT clause. To encounter this issue, both of the following conditions need to be true:
+
+- The query contains a DISTINCT clause.
```

Comment:
The queries may returns wrong results when following condition are true.

BatchNestedLoopJoin is enabled
Join clause is present in the query
DISTINCT clause is present in the query
Table has range indexes